components/x-topic-search/package.json

```
@@ -0,0 +1,41 @@
+{
+  "name": "@financial-times/x-topic-search",
+  "version": "0.0.0",
+  "description": "",
+  "main": "dist/TopicSearch.cjs.js",
+  "module": "dist/TopicSearch.esm.js",
+  "browser": "dist/TopicSearch.es5.js",
+  "style": "dist/TopicSearch.css",
+  "scripts": {
+    "prepare": "npm run build",
```

Doing this should fix the CI build (and mean other devs don't have to manually run `bower install`). 

```suggestion
    "prepare": "bower install && npm run build",
```

(I only figured this out this week 😀)